src/writing-policies/spec/06-signature-verifier-policies.md

```diff
@@ -0,0 +1,147 @@
+# Signature verifier policies
+
+Kubewarden implements support for the [Sigstore](https://www.sigstore.dev/)
+standard. This allows to implement a Secure Supply Chain for your cluster.
+
+Part of that is to ensure that all container images run the in the cluster are
+signed and verified, proving that they come from their stated authors, and
```

Comment:
```suggestion
signed proving that they come from their stated authors. With this we can also verify that they
```

Reply:
I would prefer to leave it as it is: having something signed doesn't prove that it comes from their stated authors until you verify the signature. Hence the "signed and verified" altogether.